binding/python/util.py

```diff
@@ -104,8 +104,9 @@ def pv_library_path(relative):
     raise NotImplementedError('Unsupported platform.')
 
 
-def pv_model_path(relative):
-    return os.path.join(os.path.dirname(__file__), relative, 'lib/common/porcupine_params.pv')
+def pv_model_path(relative, lang):
```

Comment:
be careful when changing the binding API - this function is used to find the path to the model file that we package with the Porcupine Python SDK (see `__init__.py`).  For the unit tests, I'd recommend using relative paths and avoid modifying the binding.